osgp/platform/osgp-core/src/main/resources/osgp-core.properties

```diff
@@ -219,7 +219,9 @@ quartz.scheduler.thread.count=1
 
 scheduling.scheduled.tasks.cron.expression=0 */1 * * * ?
 
-scheduling.task.page.size=100
+scheduling.task.page.size=1000
+scheduling.task.thread.pool.size=10
+scheduling.task.pending.duration.max.seconds=28800
```

Comment:
Waar is deze waarde op gebaseerd dan? Zou dit niet gewoon een uur moeten blijven?

Reply:
Deze waarde heb ik overgenomen uit prd settings